includes/options-page.php

```diff
@@ -285,6 +315,30 @@ function edac_simplified_summary_position_cb() {
 	<?php
 }
 
+/**
+ * Renders radio inputs for the frontend highlighter position option.
+ *
+ * @return void
+ */
+function edac_frontend_highlighter_position_cb() {
+	$position = get_option( 'edac_frontend_highlighter_position', 'right' );
+	?>
+		<fieldset>
+			<label>
+				<input type="radio" name="<?php echo 'edac_frontend_highlighter_position'; ?>" id="edac_frontend_highlighter_position" value="right" <?php checked( $position, 'right' ); ?>>
```

Comment:
Maybe the name attribute doesn't need to be PHP? If it does it should be escaped.

```suggestion
				<input type="radio" name="<?php echo esc_attr( 'edac_frontend_highlighter_position' ); ?>" id="edac_frontend_highlighter_position" value="right" <?php checked( $position, 'right' ); ?>>
```

Reply:
Good catch, this doesn't need to be in PHP at all. Will inline it just. I copied this pattern over from other callbacks though. I will make a separate PR to fix those as well.